blocks/columns/columns.css

```diff
@@ -153,6 +153,16 @@
   max-width: 1200px;
 }
 
+.columns.esg-view > div {
```

Comment:
I understand where the `esg` comes from, but will an author in the future know what this is referring to? Could we maybe use a more generic name?

Reply:
Changed the view name to **overlap**, which will work with the existing **text-and-image-in-column** view only.